examples/aws/packages/lambda-outbound-proxy/index.js

```diff
@@ -0,0 +1,24 @@
+const handler = async () => {
+  try {
+    const result = await fetch('http://ipv4.icanhazip.com/icanhazip.com');
+    const resultText = await result.text();
+    return {
+      statusCode: 200,
+      body: JSON.stringify({ msg: 'ok', outboundIp: resultText }),
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
Maybe there's an extra reason for it, but why wouldn't we have this file under the path `.packages/lambda-...`, where the rest of them live?